.github/workflows/test-from-pypi.yml

```diff
@@ -0,0 +1,54 @@
+name: Test installation from PyPI
+
+on:
+  schedule:
+    # Run at 03:27 UTC on the 8th and 22nd of every month
+    - cron: '27 3 8,22 * *'
```

Comment:
I'm not sure what the reference is here. Nothing actionable here, just wondering.

Reply:
By "reference" you mean documentation reference?

It's following the same "crontab" format as standard Unix cron. Main docs are [here](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onschedule).  More [here](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events).

Reply:
Sorry. I mean't why at 03:27 UTC on the 8th and 22nd?

Reply:
Ah, right. No particular reference. I just wanted a time that was outside normal working hours for both UK and US, and didn't match times that might be used in other cron jobs. The 8th was chosen so that this workflow first runs on the day after the 6.2.0 release, while I might still remember to check it.